spec/lib/debts/letter_downloader_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require Rails.root.join('modules', 'claims_api', 'spec', 'support', 'fake_vbms.rb')
+
+RSpec.describe Debts::LetterDownloader do
+  let(:file_number) { '796330625' }
+  let(:letter_downloader) { described_class.new(file_number) }
```

Comment:
conventionally i think this would be called `subject`, which would make the tests easier to follow IMO